docs/reference/2-ods-api/how-to-guides/how-to-add-profiles-to-the-ed-fi-ods-api.md

```diff
@@ -175,7 +220,13 @@ ProfileDefinition column.
 :::note
 
 The following GitHub link contains source files for this Profile
-sample: [Profile Source
-Files](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/tree/v7.2/Samples/Project-Profiles-Template)
+sample:
+
+* [ODS/API
+7.2](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/tree/v7.2/Samples/Project-Profiles-Template)
+* [ODS/API
+7.1](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/tree/v7.1/Samples/Project-Profiles-Template)
+* [ODS/API
+6.2](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/tree/v6.2/Samples/Project-Profiles-Template)
```

Comment:
Another example of linking out to multiple versions of a file. But I did not try to update this article to describe when dynamic profiles were first introduced. Therefore the article is not (at least not yet) appropriate for ODS/API 5.x and 6.x.

Reply:
Sounds good! Seems like the type of change that could be made / determined during a QA review.